.github/workflows/test.yaml

```diff
@@ -1,6 +1,32 @@
 name: Testing
 on: [push, pull_request]
 jobs:
+  lint:
+    name: ESLint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Node v16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Cache Deps
+        uses: actions/cache@v2
+        with:
+          path: "~/.npm"
+          key: ${{ runner.os }}-npm-cache-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-npm-cache-
```

Comment:
This is completely useless since we use npm ci, which nukes the node_modules folder. Also, ci installs are super fast compared to normal installs